media/src/settings/Settings.svelte

```diff
@@ -240,14 +246,15 @@
 <button
     class="button"
     title="Upload Scene"
+    id="upload"
     on:click={() => {
         showSettings = false;
         showUpload = !showUpload;
     }}
 >
     <i class="fa fa-upload" />
 </button>
-<button class="button" title="Download Scene" on:click={downloadScene}>
+<button class="button" id="download" title="Download Scene" on:click={downloadScene}>
```

Comment:
We can't have two DOM elements with the same html ID on the page - these need to be unique. https://developer.mozilla.org/en-US/docs/Web/API/Element/id